storage/service_plans/resolver.go

```diff
@@ -7,17 +7,13 @@ import (
 	"github.com/Peripli/service-manager/storage"
 )
 
-func ResolveSupportedPlatformIDsForPlans(ctx context.Context, plans []*types.ServicePlan, repository storage.Repository) ([]string, error) {
+func ResolveSupportedPlatformIDsForPlans(ctx context.Context, plans []*types.ServicePlan, repository storage.Repository) (map[string]*types.Platform, error) {
```

Comment:
isn't this exactly what the other function in the file is doing? you can just call the other one when you need it, and not change the other flows using this one